srai/neighbourhoods/h3_neighbourhood.py

```diff
@@ -87,6 +87,20 @@ def get_neighbours_at_distance(self, index: str, distance: int) -> Set[str]:
         neighbours.discard(index)
         return self._select_available(neighbours)
 
+    def get_ij_index(self, anchor_index: str, target_index: str) -> Tuple[int, int]:
```

Comment:
I have prepared a dedicated function `get_local_ij_index` in the `srai.h3` module. You can just substitute it with `get_local_ij_index(anchor_index, target_index)`. But it returns normalized distance, always bound to the anchor (it subtracts the origin index, to make it 0,0 in the center), and I don't know if it will break anything in the GeoVex methodology.